src/generator/pages.clj

```diff
@@ -52,7 +52,22 @@
              [:span.icon
               [:svg {:xmlns "http://www.w3.org/2000/svg" :width "24" :height "24" :viewBox "0 0 24 24"}
                [:path {:fill "currentColor" :d "M19 0h-14c-2.761 0-5 2.239-5 5v14c0 2.761 2.239 5 5 5h14c2.762 0 5-2.239 5-5v-14c0-2.761-2.238-5-5-5zm-.139 9.237c.209 4.617-3.234 9.765-9.33 9.765-1.854 0-3.579-.543-5.032-1.475 1.742.205 3.48-.278 4.86-1.359-1.437-.027-2.649-.976-3.066-2.28.515.098 1.021.069 1.482-.056-1.579-.317-2.668-1.739-2.633-3.26.442.246.949.394 1.486.411-1.461-.977-1.875-2.907-1.016-4.383 1.619 1.986 4.038 3.293 6.766 3.43-.479-2.053 1.08-4.03 3.199-4.03.943 0 1.797.398 2.395 1.037.748-.147 1.451-.42 2.086-.796-.246.767-.766 1.41-1.443 1.816.664-.08 1.297-.256 1.885-.517-.439.656-.996 1.234-1.639 1.697z"}]]
-              [:span {:class "sr-only"} "Twitter"]])]])
+              [:span {:class "sr-only"} "Twitter"]])
+    (link-to "https://www.itewiki.fi/cartman-digital"
+             [:span.icon
+              [:svg {:xmlns "http://www.w3.org/2000/svg"  :width "24" :height "24" :viewBox "0 0 177.7 177.7" :x "0px" :y "0px" :style "enable-background:new 0 0 177.7 177.7;" :xml:space "preserve"}
+               [:st0 [:style {:type "text/css" :fill-rule "evenodd" :clip-rule "evenodd" :fill "rgb(255,255,255)"} ]]
+               [:g [:path {:fill "rgb(255,255,255)" :class "st0" :d "M84.3,129l-40-40.3L0.2,44.2L0,133.5l39.8,39.6L84.3,129L84.3,129z M93.4,48.7l40,40.3l44.1,44.4l0.2-0.2V44.2
+                                                     		L137.9,4.6L93.4,48.7z" }]
+                [:polygon  {:fill "rgb(255,255,255)" :class "st0"  :points "129,93.4 88.6,133.4 44.2,177.5 133.5,177.7 173.1,137.9 	"}]
+                [:polygon  {:fill "rgb(255,255,255)" :class "st0" :points "48.7,84.3 89,44.3 133.5,0.2 44.2,0 4.6,39.8 	"}]
+                ]
+               
+               ]
+              [:span {:class "sr-only"} "Itewiki"]]
+             )
+    
```

Comment:
For code formatting purposes try to keep empty lines to minimum. In this case, please set the closing tags to the same line.